src/app/add-item/add-item.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="container dialogModal">
+    <h2 class="dialogTitle" mat-dialog-title>{{ title }}</h2>
+    <mat-dialog-content [formGroup]="dialogForm">
+        <mat-form-field>
+            <input matInput class="typedValueColor nameInput" placeholder="Action Item Name" formControlName="name" />
+            <mat-error class="invalidFeedback">
```

Comment:
**SUGGESTION:** It might be worth to also test validations of `mat-errors` with UI tests.